llvm/lib/CodeGen/TargetPassConfig.cpp

```diff
@@ -1199,11 +1199,12 @@ bool TargetPassConfig::addISelPasses() {
   }
 
   if (YkInsertStackMaps) {
-    addPass(createYkStackmapsPass(numberOfControlPoints));
+    bool SkipUnoptimisedCalls = (YkBasicBlockTracer && YkModuleClone) == false;
```

Comment:
Can we add a comment here? It wasn't immediately obvious to me what this does. Is this check the same as `!YkBasicBlockTracer || !YkModuleClone` (or `!(YkBasicBlockTracer && YkModuleClone` though personally I prefer the former)?

Why are we skipping unoptimisedcalls in stackmaps? I assumed we would be skipping the optimised calls as we don't trace those.